app/frontend/src/features/dashboard/Home.tsx

```diff
@@ -0,0 +1,120 @@
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  makeStyles,
+  Typography,
+} from "@material-ui/core";
+import Divider from "components/Divider";
+import { ExpandMoreIcon } from "components/Icons";
+import Markdown from "components/Markdown";
+import PageTitle from "components/PageTitle";
+import {
+  CONTRIBUTE_MARKDOWN,
+  LANDING_TEXT,
+  LAST_UPDATE,
+  UPDATES_MARKDOWN,
+  UPDATES_TITLE,
+  WEEKLY_EVENTS_MARKDOWN,
+  WEEKLY_EVENTS_SUBTITLE,
+  WEEKLY_EVENTS_TITLE,
+  WELCOME,
+} from "features/dashboard/constants";
+import DashboardBanners from "features/dashboard/DashboardBanners";
+
+import ContributorForm, { CONTRIBUTE, JOIN_THE_TEAM } from "../contributorForm";
+import MyCommunities from "./MyCommunities";
+
+const useStyles = makeStyles((theme) => ({
+  accordion: {
+    marginBlockStart: theme.spacing(2),
+    "& .MuiAccordionSummary-content": {
+      alignItems: "baseline",
+      flexWrap: "wrap",
+    },
+  },
+  accordionTitle: {
+    flexBasis: "30vw",
+    flexGrow: 1,
+    marginInlineEnd: theme.spacing(4),
+  },
+  accordionSubtitle: {
+    ...theme.typography.h3,
+    color: theme.palette.grey[600],
+    flexGrow: 100,
```

Comment:
Is there a better way to do this? I want the subtitles to all be aligned, but I also want the title to fill the entire space if the subtitle is pushed to the next line.

Reply:
I think setting the `flexBasis` for both the title and subtitle to `50%` and remove the `flexWrap` on the `accordion` class would work? That'd also allow us to avoid more "magic values" like "30vw" on the title.

Might want to reduce the `marginInlineEnd` on smaller screen with that clamp trick you showed me? The gap looks a bit too wide to me on those screens otherwise.

Reply:
I do want it to wrap, though!

Reply:
You mean for the title and subtitle to wrap without them keep going to the rest of the line right? I think what I suggested works with that too

Reply:
What I want is:
-if there's enough room for both on one line when the title width is 30vw, but them both in the same line (and have the title take up 30vw, subtitle takes the rest)
-if not, have title and subtitle on their own line, taking up the whole line

Reply:
Wouldn't we end up with cases where one long title would put things onto two lines, but the other accordion headers with a short title are still on one line with two columns if we do the wrap thing? So then the titles and subtitles are not lined up vertically anymore in those cases

Reply:
Yes, but I think it's fine?

Reply:
Personally I think it looks better if it's either all titles/subtitles are in two columns at some large screen sizes, and goes to two rows in smaller screens. Having a mix where each titles do their own thing doesn't feel as tidy to me. Maybe show a screenshot of that scenario and see what design thinks too?

Reply:
Fiiiine I'll do a media query 😠